circle.yml

```diff
@@ -0,0 +1,23 @@
+---
+machine:
+  pre:
+    - curl -sSL https://s3.amazonaws.com/circle-downloads/install-circleci-docker.sh | bash -s -- 1.10.0
+  environment:
+    IMAGE_NAME: 'zappi/oauth2_proxy'
+  services:
+    - docker
+
+test:
+  pre:
+    - docker build --tag=$IMAGE_NAME .
+  override:
+    - docker run $IMAGE_NAME --version
+
+deployment:
+  tagged:
+    tag: /.*/
+    commands:
+      - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
+      - docker tag $IMAGE_NAME:latest $IMAGE_NAME:$CIRCLE_TAG
```

Comment:
Done in 7619ae6. However, it would have still worked. Docker defaults to `latest` when no tag set.